ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRao.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.search_tree_rao;
+
+import com.farao_community.farao.data.crac_api.Crac;
+import com.farao_community.farao.ra_optimisation.RaoComputationResult;
+import com.farao_community.farao.rao_api.RaoParameters;
+import com.farao_community.farao.rao_api.RaoProvider;
+import com.google.auto.service.AutoService;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.network.Network;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+@AutoService(RaoProvider.class)
+public class SearchTreeRao implements RaoProvider {
+
+    @Override
+    public String getName() {
+        return "Search Tree Rao";
+    }
+
+    @Override
+    public String getVersion() {
+        return "1.0.0";
+    }
+
+    @Override
+    public CompletableFuture<RaoComputationResult> run(Network network, Crac crac, String variantId, ComputationManager computationManager, RaoParameters parameters) {
+        return null;
```

Comment:
Add a todo to have it listed in your IDE

Reply:
done